src/App.vue

```diff
@@ -254,7 +258,14 @@ export default defineComponent({
                 if (status.mode === 'prompt' && jsonError.value.length) {
                     toast.error(jsonError.value, '😠');
                 } else {
-                    toast.notify(status);
+                    const options: ToastOptions = {...status};
+                    if(jsx.value){
+                        options.body = h('div', null, [
+                            h('p', null, 'This is a JSX element. HTML in the body text will be escaped.'),
+                            h('p', null, status.body)
+                        ]);
+                    }
+                    toast.notify(options);
```

Comment:
I am not sure why but the below doesn't seem to trigger, if it's a jsx body with mode being loader and with barkdrop enabled

Reply:
Looks like that was at-fault of me mistakenly applying `checkIfLoading` to the jsx toggle. `checkIfLoading` also appears to do the exact opposite of what I expected - If there is even a single toast active, then withBackdrop gets set to false.
It gets even more confusing because there is no real concept of `controlled` components in vue - so if you set `withBackdrop` to `false` at the same time as it was set to true, then the UI and the state get out of sync, so you can actually toggle the backdrop on/off in the UI, but nothing in the state will change unless you have at least one toast open.
I think that getting rid of the `checkIfLoading` function is potentially the best bet - especially since you have it so that it shows the warning if there is a loader and backdrop anyway.

Reply:
yeah that's definitely confusing, I'm not sure what was going through my head?